app/styles/modules/_menu.scss

```diff
@@ -4,20 +4,106 @@
 #menu {
 	@include container-ext;
 	background: $color__primary--dark;
+	z-index: 2;
+	@include bp(medium) {
+		position: fixed !important;
+		width: 100%;
+		top: 0px;
+	}
 }
 
-.menu {
+#toggleWrapper {
+	display: none;
+	@include bp(medium) {
+		display: block;
+		height: auto;
+		background: $color__primary--dark;
+	}
+}
+
+.menu-wrapper {
+	@include container-int;
+	@include bp(medium) {
+		height: auto;
+		flex-direction: column;
+	}
+}
+
+#menuItems {
 	@include container-int;
+	@include bp(medium) {
+		flex-direction: column;
+		max-width: $menu-width--resp;
+		position: fixed;
+		background: $color__primary--dark;
+		left: - $menu-width--resp;
+		transition: left 0.4s;
+		height: 100%;
+	}
 }
 
 .menu__item{
+	display: block;
 	list-style:none;
 	font-size: $font__giant;
 	color: $color__heading--menu;
-	padding: 20px;
+	padding: $menu__item-padding--top-bottom $menu__item-padding--left-right;
+	transition: all 0.2s;
+	@include bp(medium) {
+		padding-left: 40px;
+	}
 }
 
 .menu__item:hover{
 	color:$color__primary;
 	background:$color__heading--menu;
-}
\ No newline at end of file
+}
+
+/* toggle visibility of menu on smalles screens */
+
+#menuToggleBox {
+	display: none !important;
+}
+
+#menuToggle {
+	display: none;
+
+	@include bp(medium) {
+		@include border-radius(5px);
+		background: $color__primary--soften;
+		cursor: pointer;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: space-around;
+		padding: 6px 0;
+		height: $menu__toggle-button--size;
+		width: $menu__toggle-button--size;
+		margin-left: 10px;
+		transition: justify-content 0.2s;
+	}
+}
+
+.menuToggle__stripe {
+	height: $menuToggleBox__stripe--height;
```

Comment:
if you are trying to keep with BEM naming, --height does not seem to be a modifier